src/buffer.hh

```diff
@@ -249,14 +252,17 @@ private:
         StringView back() const { return BufferLines::back()->strview(); }
     };
     LineList m_lines;
+    ByteOrderMark m_byte_order_mark = ByteOrderMark::None;
+    EolFormat m_eol_format = EolFormat::Lf;
```

Comment:
Those are out of sync with the options, and only represent the state of the buffer at last reload, I am not keen on storing that.